### ISSUE TITLE
Friends list is used for name auto-completion, even in contexts where it doesn't make sense

### ISSUE THREAD

**Issue by stevecotton** [wesnoth/wesnoth#3199]
For example, the `:control` command's auto-complete includes the friends list, even though the only arguments that make sense are the players in the current game.

**Comment by ProditorMagnus** [comment]
Also, I believe players own name is not completed, while that would make sense for control.

**Comment by encIave** [comment]
I agree, this is very annoying.. I know there are people opposite of that who are happy with such "unwanted for majority" behavior, people who watch games instead of playing and like to chat a lot externally. But there are few of them. For these special people there could be options in settings to enable friend list autocompilation. For most I believe it's a nightmare 'feature". I want autocompile only players in game, NOT external players, debug commands or friends etc..

**Comment by ProditorMagnus** [comment]
I disagree, completing commands is nice, though it could work like zsh complete instead of like bash complete. (first tab: show options, next tabs: go to next option). It does make sense to not complete debug commands unless debug is active.
